setup.cfg

```diff
@@ -20,8 +20,6 @@ force_grid_wrap=0
 combine_as_imports=True
 line_length=88
 profile=black
-skip=
```

Comment:
With the new `profile=black` option, skipping `docs/source/conf.py` is no longer necessary.